blueprints/component-test/files/tests/__testfolder__/__path__/__test__.js

```diff
@@ -0,0 +1,19 @@
+/* jshint expr:true */
+import { expect } from 'chai';
+import {
+  describeComponent,
+  it
+} from 'ember-mocha';
+
+describeComponent(
+  '<%= dasherizedModuleName %>',
+  '<%= testPrefix %><%= classifiedModuleName %>Component',
+  {
+    <%= testOpts %>
+  },
+  function() {
+    it('renders', function() {
+      <%= defaultTest %>
```

Comment:
I don't love that this information is now in `index.js`, but lodash templating is very simple and produces extremely ugly templates and output when using their if statement.

None of the ember-cli templates make use of the `<% if (foo) { %>` style templates, I'm assuming because of the bad readability and ugly output.